pocs/utils/social_twitter.py

```diff
@@ -0,0 +1,30 @@
+import tweepy
+
+from pocs.utils.logger import get_root_logger
+
+
+class SocialTwitter(object):
+
+    """Messaging class to output to Twitter
+    """
+    logger = get_root_logger()
+
+    def __init__(self, consumer_key, consumer_secret, access_token, access_token_secret, output_timestamp):
+        # Create a new twitter api object
+        try:
+            auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+            auth.set_access_token(access_token, access_token_secret)
+
+            self.api = tweepy.API(auth)
+            self.output_timestamp = output_timestamp
+        except tweepy.TweepError as e:
```

Comment:
Does this error mean that the object is unusable? If so, it would probably be best to raise the exception again.